charts/fleet-agent/README.md

```diff
@@ -1,3 +1,8 @@
-Standalone Fleet users use this chart for agent-based registration. For more details see [agent initiated registration](https://fleet.rancher.io/agent-initiated).
+## Fleet Agent Helm Chart
+
+Every Fleet managed downstream cluster will run an agent that communicates back to the Fleet manager. This agent is just another set of Kubernetes controllers running in the downstream cluster.
```

Comment:
nit (isn't the Fleet _manager_ commonly referred to as the Fleet _controller_?):
```suggestion
Every Fleet-managed downstream cluster will run an agent that communicates back to the Fleet controller. This agent is just another set of Kubernetes controllers running in the downstream cluster.
```